src/sentry/incidents/models.py

```diff
@@ -106,7 +108,7 @@ class Incident(Model):
     identifier = models.IntegerField()
     # Identifier used to match incoming events from the detection algorithm
     detection_uuid = UUIDField(null=True, db_index=True)
-    status = models.PositiveSmallIntegerField(default=IncidentStatus.OPEN.value)
+    status = models.PositiveSmallIntegerField(default=IncidentStatus.CLOSED.value)
```

Comment:
Does this need a migration or does django set this default value in the ORM?

Reply:
This might need a migration. It probably won't do anything other than update the model state

Reply:
The default seems arbitrary. Do we have to define it?

Reply:
No, it just made sense in the open/closed model, you can remove it here though. You'll need to generate a migration.